src/components/albums/styles.module.css

```diff
@@ -0,0 +1,118 @@
+.album_container {
+  display: flex;
+  flex-direction: column;
+  margin-inline: 1.5rem;
+}
+.album_content {
+  max-width: 75rem;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 2rem 0.93rem;
```

Comment:
quando for quebrado assim tipo esse 0.93 pode converter pro mais próximo, ou 1rem ou 0.875rem

Reply:
alteração realizada commit: [bbb3a23](https://github.com/tech-pro-bem/abre/pull/43/commits/bbb3a23bddea3dd3eaee0f7034c15e26f9b6eb80)